src/types.js

```diff
@@ -19,18 +79,46 @@ export const TYPES = {
     name: 'Community Guidelines',
     fileName: 'community_guidelines'
   },
+  customerPrivacyPolicy: {
```

Comment:
In favor of merging with `privacy`, unless I misunderstand and there is a difference.